src/SourceBuild/tarball/content/src/source-mappings.json

```diff
@@ -0,0 +1,152 @@
+{
+    "defaults": {
+        "defaultRef": "main",
+        "exclude": [
+            "**/*.dll",
+            "**/*.Dll",
+            "**/*.exe",
+            "**/*.pdb",
+            "**/*.mdb",
+            "**/*.zip",
+            "**/*.nupkg"
+        ]
+    },
+    "mappings": [
+        {
+            "name": "arcade",
```

Comment:
What is the experience if a new repo is added to the graph (via versions.details.xml) but not source-mapping exists?

Reply:
An exception during the MSBuild task:
`No mapping found named "XYZ"`